cpp/src/parquet/column_reader.h

```diff
@@ -367,6 +368,7 @@ class PARQUET_EXPORT RecordReader {
   uint8_t* values() const { return values_->mutable_data(); }
 
   /// \brief Number of values written including nulls (if any)
+  /// If read_dense_for_nullable_ is true this will not include nulls.
```

Comment:
nit: please try to phrase things relative to publicly visible APIs.  E.g. if this Reader was constructed with read_dense_for_nullable set to true.  I think rephrasing the `\brief` a little bit could make sense since we essentially contradict (including nulls) if any on the very next line.

Reply:
Done.